code/modules/mob/dead/dead.dm

```diff
@@ -14,7 +14,6 @@ INITIALIZE_IMMEDIATE(/mob/dead)
 	flags_1 |= INITIALIZED_1
 	// Initial is non standard here, but ghosts move before they get here so it's needed. this is a cold path too so it's ok
 	SET_PLANE_IMPLICIT(src, initial(plane))
-	tag = "mob_[next_mob_id++]"
```

Comment:
what if this is important though?

Reply:
See I thought for a moment it might be important but it seems actually like just an error
Line was added 7 years ago in #26886 when dead mobs were made to no longer call parent in initialize
Tag assignment for mobs were moved out of initialize 6 years ago in #34512 
I believe the latter just forgot to check for the former